WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/LayoutPickerCollectionViewCell.swift

```diff
@@ -4,13 +4,8 @@ import Gridicons
 
 class LayoutPickerCollectionViewCell: UICollectionViewCell {
 
-    static var cellReuseIdentifier: String {
-         return "LayoutPickerCollectionViewCell"
-     }
-
-    static var nib: UINib {
-        return UINib(nibName: "LayoutPickerCollectionViewCell", bundle: Bundle.main)
-    }
+    static let cellReuseIdentifier = "LayoutPickerCollectionViewCell"
+    static let nib = UINib(nibName: "LayoutPickerCollectionViewCell", bundle: Bundle.main)
```

Comment:
This is directly related to these changes but I noticed it was inconsistent with other files in this group so I went ahead and updated it here